www/artists/dloran/artist.json

```diff
@@ -3,5 +3,6 @@
   "location": "Regina",
   "changelog": [
     "Created by Mark Romanow: 1996."
```

Comment:
The build is failing because you need a comma at the end of this line.

Reply:
What line would I put the comma on? I always have this problem when I edit the `artist.json` file.

Reply:
The line I commented on, line 5.